### ISSUE TITLE
S4426 FN: On .Net 7 when using ECDsaOpenSsl

### ISSUE THREAD

**Issue by costin-zaharia-sonarsource** [SonarSource/sonar-dotnet#6164]
### Description

The rule S4426 does not raise issues for `ECDsaOpenSsl` when running on .Net 7 runtime.

### Repro steps

```
var ec3 = new ECDsaOpenSsl();
ec3.GenerateKey(ECCurve.NamedCurves.brainpoolP192t1); // Noncompliant {{Use a key length of at least 224 bits for EC cipher algorithm.}}
```

### Expected behavior

An issue should be raised.

### Actual behavior

No issue raised.

### Known workarounds

Please provide a description of any known workarounds.
